composer.json

```diff
@@ -185,6 +187,12 @@
             },
             "drupal/publication_date": {
                 "Issue #2983348: Published on empty for existing nodes.": "patches/publication_date-fill_published_at-2983348-60-for-composer.patch"
+            },
+            "drupal/pathologic": {
+                "Issue #758118: wild card for Additional paths.": "patches/pathologic-758118-wildcard-for-additional-paths-18.patch"
+            },
+            "drupal/linkit": {
+                "Issue #2877535: Link shown after the autocomplete selection is the bare node/xxx link, not the alias": "patches/linkit-2877535-link-shown-after-autocomplete-is-bare-41.patch"
```

Comment:
This patch is for the link browser.  The default behaviour is to show the internal url to the user after they have selected a link, e.g. `/node/123`.  But our users won't know about these types of urls, and only know the `content/123` urls, so the patch updates the link browser to show them this instead.